src/overlay.py

```diff
@@ -133,21 +151,58 @@ def toggle_size(self):
         self.is_minimized = not self.is_minimized
         if self.is_minimized:
             self.hide_canvas(None)
-            self.toggle_button.config(text="max")
+            self.toggle_button.config(text="Max")
         else:
             self.show_canvas(None)
-            self.toggle_button.config(text="min")
+            self.toggle_button.config(text="Min")
         win_spec = WindowSpec(IniConfigLoader().advanced_options.process_name)
         move_window_to_foreground(win_spec)
 
-    def filter_items(self, force_refresh=False):
+
+    def move_items(self):
+        self.toggle_button.grid_remove()
+        self.filter_button.grid_remove()
+        self.start_scripts_button.grid_remove()
+        self.move_button.grid_remove()
+
+        self.items_to_inventory_button.grid(row=1, column=0, padx=5, sticky="e")
+        self.items_to_stash_button.grid(row=1, column=3, padx=5, sticky="w")
+        
+    def filter_items(self):
```

Comment:
can you rename this to something more telling. filter_items in a loot filter program is very misleading